whisper/whisperv5/whisper.go

```diff
@@ -82,6 +83,9 @@ func NewWhisper(server MailServer) *Whisper {
 		Run:     whisper.HandlePeer,
 	}
 
+	seed := time.Now().Unix()
+	mrand.Seed(seed)
```

Comment:
Please leave this out. Seeding the global RNG can be done by the code that sets up whisper.